### ISSUE TITLE
Midterm Review (an533)

### ISSUE THREAD

**Issue by AnthonyNiznik** [andrewkozma/4741-Project#10]
The project is about analyzing NBA basketball shot data in order to identify the best situations for taking a shot. The data includes details about every shot made and also every game in the 2014 - 2015 season.

Likes
The mean squared error graph with polynomial orders is very good since we can visually see why a higher order does not necessarily lead to better fit of the true model. I also enjoyed seeing you plot shooter percentage vs defender distance. This is definitely an important relationship (though it would be nice to see actual data on the plot too). It is also good that you analyzed certain data that you thought had an underlining relationship (i.e. shooting percentage vs time on the clock).

Improvements
Here are a few statements made in the midterm report that I encourage you to look further into. For your test set, you chose to use the data from every fifth game of the season. The test set would then contain data from games that occurred at a similar time of the year (or under different conditions than the other games) which could mean the distribution of the training set is not the same as the distribution of the test set (you should check this out). You should probably randomly choose from all games to be in your training and testing set.

It would be good to refresh the reader why you are analyzing the data in the first paragraph ("identify the best situations for taking a basketball shot"). Also in your next steps section, you should be careful of how you define "skill level" of a player. If you use player ranking, make sure you say that you are comparing rank to shooting percentage and not skill level.
